README.md

```diff
@@ -1,23 +1,35 @@
-Puppet PowerShell DSC Module
-============================
+# dsc
+[wmf-5.0]: https://www.microsoft.com/en-us/download/details.aspx?id=46889
+[DSCResources]: https://github.com/powershell/DSCResources
 
-## Overview
+
+#### Table of Contents
+1. [Module Description - What is the dsc module and what does it do](#module-description)
+2. [Prerequisites](#windows-system-prerequisites)
+3. [Setup](#setup)
+4. [Usage](#usage)
+  * [Install Website Example](#install-website-example)
+5. [Limitations](#limitations)
+6. [Notes](#notes)
+7. [License](#license)
+
+## Module Description
 
 Puppet module for managing Windows poweshell DSC resources.
```

Comment:
You should fix the misspelling here while you are at it. Also, somewhere in here, it would be nice to spell out the acronym (DSC).